_rules/aria-required-context-role-ff89c9.md

```diff
@@ -211,11 +211,9 @@ This `section` element with an [explicit role][] of `doc-biblioentry` has a role
 ```html
 <section role="doc-bibliography">
 	<h1>Cited Works</h1>
-	<div role="list">
-		<p role="doc-biblioentry" id="b8cab5dd-bc24-459c-9858-7afa9da69b64">
-			John Steinbeck, The Grapes of Wrath (New York: The Viking Press, 1939)
-		</p>
-	</div>
+	<p role="doc-biblioentry" id="b8cab5dd-bc24-459c-9858-7afa9da69b64">
+		John Steinbeck, The Grapes of Wrath (New York: The Viking Press, 1939)
+	</p>
```

Comment:
This should not be changed. The point of this example is to demonstrate that "owned by" as used in ARIA means any descendant, not just the child. This example comes directly from the DPUB ARIA spec:

https://www.w3.org/TR/dpub-aria-1.0/#doc-biblioentry

This confusion about "owned by" including all descendants, and not just the children and direct references of aria-owns is a known issue in ARIA. See: w3c/aria#1161, w3c/aria#748.

Reply:
🙈 That example has so many problems I want to unsee it…

Fortunately (maybe), `doc-biblioentry` is deprecated in DPUB-ARIA 1.1 (https://w3c.github.io/dpub-aria/#doc-biblioentry) precisely because of the wonkyness it creates with the `list` role (https://github.com/w3c/dpub-aria/issues/15 and https://github.com/w3c/dpub-aria/pull/22). This is very recent changes!

I guess this example should be completely revamped. As far as I can tell, there is no more required context roles in DPUB-ARIA 1.1. I don't think it is a good idea to keep an example showing a known problem in the specs which is already fixed in the next version.

If the point of the example is to show the "descendant" meaning of "owned by", we need to find a non DPUB example.

If the point of the example is to show that DPUB roles are ignored, we can take any, but they have no required context role…

Maybe we should update the Background note saying that DPUB-ARIA 1.1 does not have any required context role (like Graphics ARIA) which justifies our choice to restrict ourselves to ARIA 1.1 roles.